### DISCUSSION TITLE
Reload of Webpage after reboot

### DISCUSSION THREAD

**Discussion by friedpa** [jomjol/AI-on-the-edge-device#1120] [General] [answered]
After clicking on reboot, the Webpage will get reloaded after 25 sec. Both of my Systems are after 25 sec not ready, would it therefore be better to extend this timeframe to e.g. 60 sec? Would not harm anyone, but the first taken picture would be for sure displayed. Only a cosmetic issue anyway ....

**Comment by caco3** [comment]
The web page calls the webserver once per second. When it is responding again, it is treated as rebooted and the page gets reloaded.
That's the easiest way to do it.
I agree that it would nicer to have a better feedback, but that is tricky.

did it effectively take longer for your systems until the page reloaded?
Then I agree we simply could increase the value.

**Comment by friedpa** [comment]
Sorry for my late reply.
No, it did not take any longer as before. I think the 25sec are a bit too short.

**Comment by caco3** [comment, marked as answer]
improved in https://github.com/jomjol/AI-on-the-edge-device/pull/1169
